static/app/components/metrics/chart/chart.tsx

```diff
@@ -142,11 +142,11 @@ export const MetricChart = memo(
         }
       });
 
-      const bucketSize = series[0]!?.data[1]!?.name - series[0]!?.data[0]!?.name;
```

Comment:
This is a good example where a manual fix was needed...

Before we had `!`  next to each of the optional fields, chained all the way down.

Now we have one `!` at the end of the chain.